examples/k8s/simple_psat/spire-server.yaml

```diff
@@ -13,6 +13,40 @@ metadata:
 
 ---
 
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: spire
+  name: spire-server-role
+rules:
+- apiGroups: [""] # "" indicates the core API group
+  resources: ["pods"]
+  verbs: ["get"]
+- apiGroups: ["authentication.k8s.io"]
+  resources: ["tokenreviews"]
+  verbs: ["get", "create"]
+- apiGroups: ["authentication.k8s.io"]
+  resources: ["subjectaccessreviews "]
```

Comment:
i didn't see this api being used directly.. is this required?

Reply:
You are right, it is not required. I'll remove it :+1: